README.md

```diff
@@ -86,6 +83,17 @@ Contribute
   automatically closes the issue if your pull request gets merged.
 
 
+## Branches
+
+The repository has the following permanent branches:
+
+ * **master** This contains the code which has been released.
+
+ * **develop** This contains the latest code. All the contributing PRs must be sent to this branch. When we want to release the next version of the app, this branch is merged into the `master` branch.
+
+ * **aws** 
+
```

Comment:
This is the branch through which the project is deployed

Reply:
I added it.